<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>471</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string>Segmentation</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="mothres_button">
       <property name="text">
        <string>Multi-Otsu Thresholding</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cvsegmen_button">
       <property name="text">
        <string>Chan-Vese Segmentation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="morphosnake_button">
       <property name="text">
        <string>Morphological Snakes</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>321</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>qt_lab_iconlar/save.png</normaloff>qt_lab_iconlar/save.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_as_button">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>qt_lab_iconlar/save-as.jpg</normaloff>qt_lab_iconlar/save-as.jpg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="export_as_button">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>qt_lab_iconlar/export_as.png</normaloff>qt_lab_iconlar/export_as.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="undo_button">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>qt_lab_iconlar/undo_icon.png</normaloff>qt_lab_iconlar/undo_icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="redo_button">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>qt_lab_iconlar/redo.png</normaloff>qt_lab_iconlar/redo.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_button">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>qt_lab_iconlar/clear.png</normaloff>qt_lab_iconlar/clear.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>595</width>
      <height>470</height>
     </rect>
    </property>
    <property name="title">
     <string>Source</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>622</x>
      <y>150</y>
      <width>595</width>
      <height>470</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string>Conversion</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="rgb2gray_button">
       <property name="text">
        <string>RGB TO GRAY</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rgb2hsv_button">
       <property name="text">
        <string>RGB TO HSV</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>241</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string>Edge Dedection</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="roberts">
       <property name="text">
        <string>Roberts</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sobel">
       <property name="text">
        <string>Sobel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scharr">
       <property name="text">
        <string>Scharr</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prewitt">
       <property name="text">
        <string>Prewitt</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>110</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string>Source</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="file">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>qt_lab_iconlar/file.png</normaloff>qt_lab_iconlar/file.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="export_as">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>qt_lab_iconlar/export.png</normaloff>qt_lab_iconlar/export.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>qt_lab_iconlar/clear.png</normaloff>qt_lab_iconlar/clear.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1240</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport_As">
     <property name="title">
      <string>Export As</string>
     </property>
     <addaction name="actionSource"/>
     <addaction name="actionOutput"/>
    </widget>
    <addaction name="actionOpen_Source"/>
    <addaction name="actionSave_Output"/>
    <addaction name="actionSave_As_Output"/>
    <addaction name="menuExport_As"/>
    <addaction name="actionExit_2"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuClear">
     <property name="title">
      <string>Clear</string>
     </property>
     <addaction name="actionSource_2"/>
     <addaction name="actionOutput_2"/>
    </widget>
    <addaction name="menuClear"/>
    <addaction name="actionUndo_Output"/>
    <addaction name="actionRdeo_Output"/>
   </widget>
   <widget class="QMenu" name="menuconversion">
    <property name="title">
     <string>conversion</string>
    </property>
    <addaction name="actionRGB_to_Grayscale"/>
    <addaction name="actionRGB_to_HSV"/>
   </widget>
   <widget class="QMenu" name="menuSegmentation">
    <property name="title">
     <string>Segmentation</string>
    </property>
    <addaction name="actionMulti_Otsu_Thresholding"/>
    <addaction name="actionChan_Vese_Segmentation"/>
    <addaction name="actionMorphological_Snakes"/>
   </widget>
   <widget class="QMenu" name="menuMulti_Otsu_Thresholding">
    <property name="title">
     <string>Multi-Otsu Thresholding</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdge_Detection">
    <property name="title">
     <string>Edge Detection</string>
    </property>
    <addaction name="actionRoberts"/>
    <addaction name="actionSobel"/>
    <addaction name="actionScharr"/>
    <addaction name="actionPrewitt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuconversion"/>
   <addaction name="menuSegmentation"/>
   <addaction name="menuMulti_Otsu_Thresholding"/>
   <addaction name="menuEdge_Detection"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Source">
   <property name="text">
    <string>Open Source</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_Output">
   <property name="text">
    <string>Save Output</string>
   </property>
  </action>
  <action name="actionSave_As_Output">
   <property name="text">
    <string>Save As Output</string>
   </property>
  </action>
  <action name="actionExport_as">
   <property name="text">
    <string>Export as</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSource">
   <property name="text">
    <string>Source</string>
   </property>
  </action>
  <action name="actionOutput">
   <property name="text">
    <string>Output</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Shift+F4</string>
   </property>
  </action>
  <action name="actionSource_2">
   <property name="text">
    <string>Source</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionOutput_2">
   <property name="text">
    <string>Output</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
  <action name="actionUndo_Output">
   <property name="text">
    <string>Undo Output</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionRdeo_Output">
   <property name="text">
    <string>Redo Output</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionRGB_to_Grayscale">
   <property name="text">
    <string>RGB to 
Grayscale</string>
   </property>
  </action>
  <action name="actionRGB_to_HSV">
   <property name="text">
    <string>RGB to HSV</string>
   </property>
  </action>
  <action name="actionMulti_Otsu_Thresholding">
   <property name="text">
    <string>Multi-Otsu Thresholding</string>
   </property>
  </action>
  <action name="actionChan_Vese_Segmentation">
   <property name="text">
    <string>Chan-Vese Segmentation</string>
   </property>
  </action>
  <action name="actionMorphological_Snakes">
   <property name="text">
    <string>Morphological Snakes</string>
   </property>
  </action>
  <action name="actionRoberts">
   <property name="text">
    <string>Roberts</string>
   </property>
  </action>
  <action name="actionSobel">
   <property name="text">
    <string>Sobel</string>
   </property>
  </action>
  <action name="actionScharr">
   <property name="text">
    <string>Scharr</string>
   </property>
  </action>
  <action name="actionPrewitt">
   <property name="text">
    <string>Prewitt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
